src/js/letters/reducers/index.js

```diff
@@ -23,16 +27,34 @@ function letters(state = initialState, action) {
       // the various error scenarios
       return set('lettersAvailable', false, state);
     case 'GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS':
+      // Create object for which options to include in post for summary letter
+      // Default all options to true so they appear checked in the UI
+      options = mapValues(action.data.data.attributes.benefitInformation, (value) => {
+        let isSelected = false;
+        // If the value of the benefit option is not false, it means the user
```

Comment:
I like this whole comment so it's clear what's happening. I'd shorten the code to
```
return value && isSelected
```